_includes/tutorials/error-handling/kstreams/code/build.gradle

```diff
@@ -0,0 +1,67 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.22.0"
+        classpath "com.github.jengelman.gradle.plugins:shadow:6.0.0"
+    }
+}
+
+plugins {
+    id "java"
+    id "com.google.cloud.tools.jib" version "2.8.0"
+    id "idea"
+    id "eclipse"
+}
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
+version = "0.0.1"
+
+repositories {
+    jcenter()
+
+    maven {
+        url "https://packages.confluent.io/maven"
+    }
+}
+
+apply plugin: "com.commercehub.gradle.plugin.avro"
+apply plugin: "com.github.johnrengelman.shadow"
+
+dependencies {
+    implementation "org.apache.avro:avro:1.10.1"
+    implementation "org.slf4j:slf4j-simple:1.7.30"
+    //implementation "org.apache.kafka:kafka-streams:2.8.0"
+    implementation files('temp_local_28_repo/kafka-streams-2.8.0-SNAPSHOT.jar')
+     //implementation "org.apache.kafka:kafka-clients:2.8.0"
+    implementation files('temp_local_28_repo/kafka-clients-2.8.0-SNAPSHOT.jar')
+    //testImplementation "org.apache.kafka:kafka-streams-test-utils:2.8.0"
+    testImplementation files('temp_local_28_repo/kafka-streams-test-utils-2.8.0-SNAPSHOT.jar')
```

Comment:
the `files` for the local repo needs to be deleted before emerging, but are needed as 2.8 isn't in Maven Central yet.

Lines above in comments need to be un-commented before merging